fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -2731,6 +2735,27 @@ class DDTeamCollectionImpl {
 		}
 	}
 
+	ACTOR static Future<Void> serverGetStorageServerStatusRequest(DDTeamCollection* self, TeamCollectionInterface tci) {
```

Comment:
How are these serverGet*() going to be used?

Reply:
I suppose that the `DataDistributionRuntimeMonitor` may acquire SS metrics and Team healthiness to make scheduling decision.